### PR TITLE
UI Fixed: Styling of adding Related Terms and Synonyms is different #7671

### LINKED ISSUE #7671: Styling of adding Related Terms and Synonyms is different
Styling of adding Related Terms and Synonyms is different. We should provide the similar experience - P0 - 0.12.1

### LINKED ISSUE #7672: Improve Assets listing of Glossary Term - P0 - 0.12.1

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

I worked on Fixing #7671 #7672
Closes #7671 
Closes #7672
#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :

https://user-images.githubusercontent.com/71748675/191828272-53d6a40a-f32b-4d3d-a4f2-464ff949add1.mov

image
image

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/cypress.config.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default defineConfig({
     setupNodeEvents(on, config) {
       return plugins(on, config);
     },
-    baseUrl: 'http://localhost:8585',
+    baseUrl: 'http://localhost:3000',
```
[the remark this answers is not in the document]
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/cypress.config.ts:18]: Nice catch, it is not intentional

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js [hunk 2]
```diff
@@ -225,43 +225,45 @@ describe('Glossary page should work properly', () => {
...
-      .as('synonyms');
-    cy.get('@synonyms').clear();
-    cy.get('@synonyms').type(uSynonyms);
+      .click();
 
-    cy.intercept({ method: 'PATCH', url: '/api/v1/glossaryTerms/*' }).as(
-      'getGlossary'
-    );
+    cy.get('.ant-select-selector').should('be.visible');
+    cy.get('.ant-select-clear > .anticon > svg')
```
[the remark this answers is not in the document]
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Glossary.spec.js:238]: I tried that way but Antd select is not picking up data-testid or id so for this case we have to depend on class

File: openmetadata-ui/src/main/resources/ui/src/components/GlossaryTerms/tabs/GlossaryTermReferences.tsx [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+        updatedGlossaryTerm = {
+          ...updatedGlossaryTerm,
+          references: updatedRef,
+        };
+
+        onGlossaryTermUpdate(updatedGlossaryTerm);
+      }
+      setIsViewMode(true);
+    } catch (error) {
+      // Added catch block to prevent uncaught promise
```
[the remark this answers is not in the document]
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/GlossaryTerms/tabs/GlossaryTermReferences.tsx:45]: This is to catch the inline error if URL input is valid or not, antd is already showing it!

File: openmetadata-ui/src/main/resources/ui/cypress.config.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/cypress.config.ts]: Fixed

**Review by Sachin-chaurasiya — APPROVED** [review]

**Comment by chirag-madlani** [comment]
@ShaileshParmar11  @devyani-kaushik Why we have edit icon on right corner for Description and for rest of them it's just after the heading

**Comment by devyani-kaushik** [comment]
@chirag-madlani Icon placement was skipped by me, will check and get it updated.

**Pull request merged**